src/consensus/consensus.h

```diff
@@ -28,4 +30,8 @@ enum {
     LOCKTIME_MEDIAN_TIME_PAST = (1 << 1),
 };
 
+static const int64_t SECONDS_PER_YEAR = 31558060;
```

Comment:
EPOCH seconds are not calculated by years * seconds_per_year,  there are time tweak gaps, leap years, historical time change reasons etc.